PurchasesTests/Identity/IdentityManagerTests.swift

```diff
@@ -86,21 +87,22 @@ class IdentityManagerTests: XCTestCase {
         expect(self.mockBackend.invokedCreateAlias).toEventually(beTrue())
     }
 
-    func testCreateAliasNoOpsIfNilAppUserID() {
+    #if arch(x86_64)
```

Comment:
this will still run in CI right?

Reply:
That's right currently CI is Intel only.

Reply:
cool just confirming!